docs/docs/recipes/working-with-images.md

```diff
@@ -1,73 +1,75 @@
 ---
-title: "Recipes: Working with Images"
+Titre: "Recettes: travailler avec des images"
```

Comment:
Please don't change used variables `title` is used in compilation time

Reply:
Okay changed